src/main/java/org/fundacionjala/trello/pages/BoardCreation.java

```diff
@@ -25,23 +25,27 @@
     @FindBy(css = "[class='subtle-chooser-trigger unstyled-button org-chooser-trigger']")
     private WebElement selectTeamButton;
 
+    private String titleString = null;
+    private String privacyString = null;
+    private String backgroundString = null;
 
     /**
      * Method for create a dashboard with some specs.
      *
      * @param data List of the data specs.
      * @return the PO of the Selected DashBoard.
      */
-    public SelectedDashBoard createNewBoard(final Map<BoardFields, String> data) {
+    public SelectedBoard createNewBoard(final Map<BoardFields, String> data) {
         EnumMap<BoardFields, ISteps> dashboardSteps = new EnumMap<>(BoardFields.class);
-        dashboardSteps.put(BoardFields.TITLE, () -> action.setValue(titleTextInputField, data.get(BoardFields.TITLE)));
+        titleString = data.get(BoardFields.TITLE);
+        dashboardSteps.put(BoardFields.TITLE, () -> action.setValue(titleTextInputField, titleString));
```

Comment:
Dennis, this should be called board only not dashboard, please could you review it in all files?